pkg/index/inverted/query.go

```diff
@@ -49,30 +49,22 @@ type GlobalIndexError struct {
 
 func (g GlobalIndexError) Error() string { return g.IndexRule.String() }
 
+var _ index.Query = (*Query)(nil)
+
 // Query is a wrapper for bluge.Query.
 type Query struct {
```

Comment:
Do you need to expose this type since its operations are wrapped by the `index.Query`?